Leanplum-SDK/Classes/Leanplum.m

```diff
@@ -3466,7 +3466,7 @@ - (BOOL)boolNamed:(NSString *)name
 {
     if ([Utils isNullOrEmpty:name]) {
         [Leanplum throwError:@"[LPActionContext boolNamed:] Empty name parameter provided."];
-        return nil;
+        return NO;
```

Comment:
I will revert it for now. I was just trying to remove the warning. It's not good to mingle things in the same comment. Will submit a different PR. 

Just fyi returning nil to BOOL will convert to NO, so it is safe.